### ISSUE TITLE
Add repository embed image

### ISSUE THREAD

**Issue by danieldegroot2** [openstreetmap-polska/openaedmap-frontend#159]
You should be able to add a logo / embed image for the GitHub repository. See
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview

**Comment by Zaczero** [comment]
Then it could also be used as an OpenGraph website preview image (https://ogp.me/).

**Comment by danieldegroot2** [comment]
Yes, the embed for the website itself could be improved too. Currently it looks like this on Discord
("AED defibrillators are..." could also maybe be cut out from the embed description here entirely but that is something else.)

![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/67521919/6d6fda93-a1d0-4d61-a203-324889e54f91)
